server/migrations/20191010101148-0.0.9.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-param-reassign */
+const { runAsyncTasks } = require('../common/utils');
+
+const up = async db => {
+  const addEmailNotificationSettings = async () =>
+    db.collection('users').updateMany(
+      {
+        emailNotificationsFrequency: { $exists: false },
+        emailReportSentAt: { $exists: false }
+      },
+      {
+        $set: {
+          emailNotificationsFrequency: '',
```

Comment:
This is not a valid value. Valid values are one of the enumerable - please assign the default here.